en/02_Developer_Guides/01_Templates/07_Caching.md

```diff
@@ -51,3 +50,9 @@ Example:
     $CacheableContent
 <% end_cached %>
 ```
+
+## Template caching
+
+Every time a raw template file is processed, some PHP code is generated from it which is then executed to produce the final rendered result.
+
+That PHP code is stored in the filesystem so that the raw template file doesn't need to be processed again until either the cache is flushed or the raw template file is updated.
```

Comment:
```suggestion
That PHP code is cached in the filesystem so that the raw template file doesn't need to be processed again until either the cache is flushed or the raw template file is updated.
```

Reply:
Just ties it together a little better

Reply:
Done